src/main/resources/resources/logisim/strings/analyze/analyze_de.properties

```diff
@@ -207,4 +207,4 @@ badVariableBitFormError = Der Variablenname muss in der Form 'Name[i]' sein.
 badVariableColonError = Der Variablenname muss vor dem ':' erscheinen.
 badVariableIndexError = Der Variablebitindex muss eine ganze Zahl sein.
 variableFormat = Variablen müssen die Form 'Name[N..0]' haben.
-variableTooMuchBits = Variablen dürfen nicht mehr als 32 Bit breit sein.
+variableTooMuchBits = Variablen dürfen nicht mehr als 32 Bit breit sein.
```

Comment:
@MarcinOrlowski when using the `--update` function from trans-tool the last newline gets removed.

Reply:
You mean at the end of the file?

Reply:
Yes.

Reply:
This is now fixed in 2.5.1